tiatoolbox/utils/misc.py

```diff
@@ -1212,9 +1216,16 @@ def patch_pred_store(
             conversion of annotations saved at non-baseline resolution to baseline.
             Should be model_mpp/slide_mpp.
         class_dict (dict): Optional dictionary mapping class indices to class names.
+        save_dir (str or pathlib.Path): Optional Output directory to save the Annotation
+            Store results. if the save_dir is not provided, then an SQLiteStore object
+            containing Annotations for each patch is returned.
+        output_file (str): Optional file name to save the Annotation Store results.
+            if the output_file is not provided, then an SQLiteStore object
+            containing Annotations for each patch is returned.
 
```

Comment:
is there any particular reason to have both save_dir and output_file? why not just save_path, or something similar?

Reply:
I agree for a misc function we can use just one variable save_path, from engines we can pass `save_path / output_name`